numba/cuda/cudadrv/nvvm.py

```diff
@@ -363,6 +354,7 @@ def get_log(self):
     (11, 8): ((3, 5), (9, 0)),
     (12, 0): ((5, 0), (9, 0)),
     (12, 1): ((5, 0), (9, 0)),
+    (12, 2): ((5, 0), (9, 0)),
```

Comment:
This is adding 12.2 whereas this patch concerns moving the baseline to 11.2?

Reply:
Yes, since this is a one-liner to add with no changed support, I added this in.

Reply:
OK, thanks, let's leave it as-is given support isn't changing.